tobiasHe/assignments/cyclicSort/cyclicSortEasy/cyclicSortEasy.js

```diff
@@ -4,14 +4,34 @@
 
 // Write a function to sort the objects in-place on their creation sequence number in O(n) and without any extra space. For simplicity, let’s assume we are passed an integer array containing only the sequence numbers, though each number is actually an object.
 
-const cyclic_sort = function (nums) {
-  // TODO: Write your code here
-  return nums;
+const cyclic_sort = function(nums) {
+	// TODO: Write your code here
```

Comment:
Nice work. Would recommend a more detailed high level overview of your approach and also typing the solution twice to help with muscle memory.